blocks/editable/format-toolbar.js

```diff
@@ -151,17 +151,19 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label className="screen-reader-text" htmlFor="editable-format-toolbar__link-input">{ __( 'URL' ) }</label>
 						<input
 							autoFocus
 							className="editable-format-toolbar__link-input"
+							id="editable-format-toolbar__link-input"
```

Comment:
Ids should contain a unique uuid. We should consider using the `[withInstanceId](http://gutenberg-devdoc.surge.sh/?selectedKind=Higher%20Order%20Components&selectedStory=withInstanceId&full=0&down=1&left=1&panelRight=1&downPanel=storybooks%2Fstorybook-addon-knobs)` HoC